pages/index.vue

```diff
@@ -63,7 +63,10 @@
       <div class="users-section">
         <div class="section-header">
           <div class="section-label green">For Players</div>
-          <h2 class="section-tagline">Discover over 5,000 creations</h2>
+          <h2 class="section-tagline">
+            Discover over {{ $formatNumber($tag.statistics.projects) }}
```

Comment:
I'd change this to format it down so that it only has 1 significant digit. I think if it's too specific it'll look weird.

Consider: **Over 76.5k creations** vs. **Over 70,000 creations**.

when it's too specific, it sorta makes it sound less impressive. Also, I would not trim the number, show the full amount of digits, just replace all but the first with zeros.

So like:
- 2,241 -> 2,000
- 12,421 -> 10,000
- 4,142,341 -> 4,000,000

etc.